src/storage/mvcc/reader/scanner/mod.rs

```diff
@@ -146,7 +146,11 @@ impl<S: Snapshot> ScannerBuilder<S> {
         ))
     }
 
-    pub fn build_delta_scanner(mut self, from_ts: TimeStamp) -> Result<DeltaScanner<S>> {
+    pub fn build_delta_scanner(
+        mut self,
+        from_ts: TimeStamp,
+        extra_op: ExtraOp,
+    ) -> Result<DeltaScanner<S>> {
```

Comment:
How about adding a new method `fn extra_op()`?

Reply:
We only need this flag for `DeltaEntryPolicy`.